server_tests/test/clientTests.js

```diff
@@ -482,24 +474,27 @@ describe("RUST API TEST SUITE", function() {
             this.timeout(timeoutValue);
             return testClient.takePicture(sessionId)
             .then( function onSuccess (res) {
-                Comparison.oscTakePictureOutput(res);
+                validate.done(res.body, schema.names.commandTakePicture);
                 return testClient.listImages(1, true, 100);
             })
             .then( function onSuccess (res) {
-                Comparison.oscListImagesOutput(res, false, true, {entries: [{'one': 'one'}], totalEntries: 1});
+                validate.done(res.body, schema.names.commandListImages);
+                assert.equal(res.body.results.entries.length, 1)
```

Comment:
This looks like length should be `2`, and another assert is needed to check that `totalEntries` is equal to 1.